api/router/DeploymentConfigRouter.go

```diff
@@ -0,0 +1,27 @@
+package router
+
+import (
+	"github.com/devtron-labs/devtron/api/restHandler"
+	"github.com/gorilla/mux"
+)
+
+type DeploymentConfigurationRouter interface {
+	initDeploymentConfigurationRouter(configRouter *mux.Router)
+}
+
+type DeploymentConfigurationRouterImpl struct {
+	deploymentGroupRestHandler restHandler.DeploymentConfigurationRestHandler
+}
+
+func NewDeploymentConfigurationRouter(deploymentGroupRestHandler restHandler.DeploymentConfigurationRestHandler) *DeploymentConfigurationRouterImpl {
+	router := &DeploymentConfigurationRouterImpl{
+		deploymentGroupRestHandler: deploymentGroupRestHandler,
+	}
+	return router
+}
+
+func (router DeploymentConfigurationRouterImpl) initDeploymentConfigurationRouter(configRouter *mux.Router) {
```

Comment:
subrouter for orchestrator/config, should add deployment in route. also service does not aligns with the name and route, have to discuss this